@ui/ui-kit/DatePicker/__test__/Datepicker.test.tsx

```diff
@@ -0,0 +1,203 @@
+import { render, cleanup, screen, fireEvent } from '@testing-library/react'
+import { MainDatePicker } from '../DatePicker'
+describe('date picker renders correctly', () => {
+  it('date picker render', () => {
+    const { asFragment } = render(<MainDatePicker />)
+    expect(asFragment()).toMatchSnapshot()
```

Comment:
you should add all props variants of Datepicker to snapshot

Reply:
done

Reply:
look how to create only 1 testcase for all snapshots with cycle , check the Button.test